attributeChart.ts

```diff
@@ -75,18 +74,23 @@ export async function widget(
 }
 
 export function createChartData(results: any, attributes: Attribute[] = []) {
-  const labels = results.map((d: any) => d.name.replace("Journal/Day/", ""));
+  const labels = results.map((d: any) => d.name.replace(/^.*\//, ""));
   const datasets = [];
+  
+  const nestedAttribute = (obj: any, path: string) => {
```

Comment:
Thank you for making it more flexible when specifying where to look for attributes. However, this requires current users to update their fenced code block to specify the top-level attribute. My suggestion is

```typescript
   const nestedAttribute = (obj: any, path: string) => {
    let value = path.split('.').reduce((acc, key) => (acc ? acc[key] : undefined), obj);

    // If not found, look under the "attribute" key for backward compatibility
    if (value === undefined && obj?.attribute) {
      value = obj.attribute[path];
    }

    return value;
  };
```

And could you move this utility function out of the createChartData function for better readability? thank you